tsconfig.json

```diff
@@ -3,11 +3,9 @@
     "module": "commonjs",
     "lib": [
       "dom",
-      "es5",
-      "es2015.collection",
-      "es2015.iterable"
+      "es2016"
```

Comment:
there was some issues with the types due to the last typescript version, it was solved using the es2016 lib